### PR TITLE
 Problem: Ambiguous "synchronization complete" message (fix #2168)

### LINKED ISSUE #2168: Problem: Ambiguous "synchronization complete" message
**Describe the bug**
When trying it out with @brianatcrypto, syncing works fine until it reaches the block that contains transaction(s) related to the wallet.  
When we try to sync it again with different the `batch-size`,the client-cli was "fooled" and show `Synchronization complete!`
 
**Example:**
 ```client-cli sync  --name Default_HD  -b 1000
Enter authentication token:
Synchronizing: 813 / 813 [==============================================] 100.00 % 133.16/s
Synchronization complete!

Error: Connection error: Unable to connect to TQE address: 20.49.200.60:3322
```

**To Reproduce**
1. Sync the wallet;
2. The sync is running until it reaches the "problematic"  block. "`Unable to connect to TQE address ...`" error message is shown;
3. Sync it again with different `batch-size`;
4. It shows `Synchronization complete!` but infact, it is not synced.

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: display progress only in validation

### CODE REVIEW

**Comment by leejw51crypto** [comment]
for changing batch size
let's say, txblock is 28, and batch-size is 20
it will stop at block 21

if you change to batch-size to 1
it'll continue till block 27

**Review by tomtau — COMMENTED** [review]
seems ok, but @lezzokafka / @brianatcrypto may be better to comment if this resolves the problem

File: client-cli/src/command.rs [hunk 1]
```diff
@@ -853,7 +853,7 @@ impl Command {
                 } => {
                     if let Some(ref mut pb) = progress_bar {
                         if current_block_height == final_block_height {
-                            pb.finish_println("Synchronization complete!\n");
+                            pb.finish_print("Synchronization complete!");
```
Comment (tomtau) [inline client-cli/src/command.rs:856]: why no new lines?

Comment (leejw51crypto) [reply client-cli/src/command.rs:856]: println + \n makes double display of the same message , so changed to like this

**Pull request merged**